pandas/core/arrays/base.py

```diff
@@ -530,8 +540,9 @@ def astype(self, dtype, copy=True):
 
         Returns
         -------
-        array : ndarray
-            NumPy ndarray with 'dtype' for its dtype.
+        array : ArrayLike
```

Comment:
in docstrings i usually say "np.ndarray or ExtensionArray" instead of "ArrayLike".  do we have a policy on that?

Reply:
Made change in next commit.